### PR TITLE
Upgrading to Gatsby 3

### PR DESCRIPTION
**Pull request by raffazizzi**
This PR upgrades our site to Gatsby 3. Notable changes:

* replaced `gatsby-remark-sourcename` with some code in [`gatsby-node.js`](https://github.com/umd-mith/mith-static/blob/gatsby3/gatsby-node.js#L22-L32)
* switched from deprecated `gatsby-image` to `gatsby-plugin-image`. This introduced a number of changes in queries and components. *N.B.* I removed breakpoints from image queries as they didn't seem to work as expected even though `gatsby-plugin-image` [supports them](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#all-options). I didn't notice any difference in the result, but @kirstenkeister may want to double check once this PR is merged.

Possible blocker:

On build, this error pops up:
```
(node:32703) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
```

I think it's happening in a dependency, but I can't trace which one. Ultimately it doesn't affect the build so I figure we can monitor it and try to address it after merge, but I welcome thoughts/feedback on this

### CODE REVIEW

File: src/components/paginator.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import { Link } from 'gatsby'
 import React from 'react'
 
-import paginatorStyles from './paginator.module.css'
+import './paginator.module.css'
```
Comment (trevormunoz) [inline src/components/paginator.js]: @raffazizzi Not exactly sure why these changes got bundled into the Gatsby3 branch? In any case, this was a CSS module file and now it looks like it's trying to be imported as vanilla css. This is breaking the styles.

**Review by trevormunoz — COMMENTED** [review]
This branch builds fine for me. Nice to see the new image goodies. See one comment about a seemingly-unrelated CSS issue.

File: src/components/paginator.js (hunk 1 above)
Comment (trevormunoz) [reply src/components/paginator.js]: I also noted one additional build error complaining about an image it couldn't find:
```
Error downloading image: failed to process
https://dl.airtable.com/.attachments/1a6628a605e4c91142674c80321b3806/57ea260e/header_sharedhorizons.jpg
RequestError: getaddrinfo ENOTFOUND dl.airtable.com
```

Comment (raffazizzi) [reply src/components/paginator.js]: ah, good catch. Once upgraded, it didn't like to be imported as a module without an explicit export in the CSS. Let me look into it again

**Comment by kirstenkeister** [comment]
@raffazizzi The breakpoints do actually do something: they limit the number of images that are generated to only those that are indicated (and necessary), which in turn means it doesn't take as long to build the site. I'd prefer to have the breakpoints in there unless the plugin doesn't work the same with the upgrade to gatsby 3, which it sounds like it might not?

I'm happy to check this once the merge is complete and see if I can at least get the new plugin to do what it was doing before.

**Comment by raffazizzi** [comment]
Addressed @trevormunoz comment and integrated the latest changes from @kirstenkeister that just came in. For breakpoints, let's complete the switch and re-assess what we can do with the new API going forward

**Comment by edsu** [comment]
I think part of @trevormunoz's motivation for the the gatsby3 work on umd-mith/airwaves was to get some build speedups. Maybe we'll see some here?
